packages/ckeditor5-undo/src/basecommand.ts

```diff
@@ -203,6 +206,13 @@ export default abstract class BaseCommand extends Command {
 
 			// After reversed operation has been transformed by all history operations, apply it.
 			for ( const operation of reversedOperations ) {
+				console.log( 'ELO ', operation.type );
+				// Do not apply any operation on non-editable space.
+				if ( !this.editor.model.isSelectableEditable( operation.target() ) ) {
+					console.log( 'NOPE ', operation.type );
+					return;
```

Comment:
Why `return;` instead of returning `NoOperation` as I suggested in my previous review? Were there any problems with my approach?